scripts/borg.exe.spec

```diff
@@ -39,9 +49,9 @@ exe = EXE(pyz,
           debug=False,
           strip=False,
           upx=True,
-          console=True )
+          console=True)
 
-if False:
+if True:
```

Comment:
guess that should be more flexible.

Reply:
I briefly looked into this. I hoped that pyinstaller accepts a spec file and additional arguments, but this is not the case.